app/controllers/admin/community_memberships_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
   def ban
     membership = CommunityMembership.find_by_id(params[:id])
     membership.update_attributes(:status => "banned")
+    membership.update_attributes(:admin => 0) if membership.admin == 1
```

Comment:
We should move to the new syntax hash `admin: 0`, but for the sake of consistency, this is ok here.